test/doctool/test-doctool-html.js

```diff
@@ -58,11 +58,6 @@ function toHTML({ input, filename, nodeVersion, versions }) {
 // This HTML will be stripped of all whitespace because we don't currently
 // have an HTML parser.
 const testData = [
-  {
-    file: fixtures.path('sample_document.md'),
-    html: '<ol><li>fish</li><li>fish</li></ol>' +
-      '<ul><li>Redfish</li><li>Bluefish</li></ul>'
-  },
```

Comment:
Why has this been removed?

Reply:
It's being tested twice for some reason:

https://github.com/nodejs/node/blob/b23a932bfd22cc58829dd86fb9ac8e5da7caec98/test/doctool/test-doctool-html.js#L61-L65

https://github.com/nodejs/node/blob/b23a932bfd22cc58829dd86fb9ac8e5da7caec98/test/doctool/test-doctool-html.js#L105-L109